src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1493,4 +1497,8 @@
     <value>This color scheme cannot be renamed because it is included by default.</value>
     <comment>Disclaimer presented near the controls to rename the color scheme when that functionality is disabled.</comment>
   </data>
+  <data name="ColorSchemes_TextPreview.Text" xml:space="preserve">
+    <value>Text</value>

```

Comment:
Ah, does that text preview resize appropriately when the string is long?

should we just choose a rather language-agnostic "`AaBbCc...`" instead? That might even be better.

FWIW: i hope it's not read out by narrator!

Reply:
> Ah, does that text preview resize appropriately when the string is long?

<img width="867" alt="image" src="https://user-images.githubusercontent.com/11050425/187560708-3c4be4a0-6e5f-48b0-b077-31d3c400b3c0.png">

> should we just choose a rather language-agnostic "`AaBbCc...`" instead? That might even be better.

Eh. Up to you. We could also bind it to the color scheme name but then the previews wouldn't be guaranteed to align.

> FWIW: i hope it's not read out by narrator!

Hah. They weren't when you would select a color scheme, but they were "focusable" when navigating with the arrow keys with Narrator. Just fixed it.